gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/DagManagerUtils.java

```diff
@@ -433,17 +431,17 @@ public static  JobStatus pollFlowStatus(Dag<JobExecutionPlan> dag, JobStatusRetr
   /**
```

Comment:
nit: document that `Optional.absent()` indicates job not found (e.g. rather than throwing)